content/develop/connect/insight/rdi-connector.md

```diff
@@ -0,0 +1,158 @@
+---
+categories:
+- docs
+- develop
+- stack
+- oss
+- rs
+- rc
+- oss
+- kubernetes
+- clients
+description: Connect to RDI from Redis Insight, configure pipelines, and more.
+linkTitle: RDI in Redis Insight
+stack: true
+title: RDI in Redis Insight
+weight: 4
+---
+
+Redis Data Integration (RDI) and its [ingest pipeline capability]({{< relref "/integrate/redis-data-integration/ingest" >}}) is an end-to-end solution for mirroring your application's primary database in Redis using a capture data change mechanism, and it includes a stream processor to map and transform source data such as relational tables into fast Redis data structures that match your use cases.
+You can read more about RDI's ingest architecture [on these pages]({{< relref "/integrate/redis-data-integration/ingest/architecture" >}}).
+
+As of version `2.54.0`, Redis Insight includes RDI connectivity, which allows you to connect to an [RDI management plane]({{< relref "/integrate/redis-data-integration/ingest/architecture" >}}#how-rdi-is-deployed), create, test, and deploy [RDI pipelines]({{< relref "/integrate/redis-data-integration/ingest/data-pipelines/data-pipelines" >}}), and view RDI statistics.
+
+## Connect
+
+Open Redis Insight, click on the **Redis Data Integration** tab, and then click on one of the two **+ Add RDI Endpoint** buttons as shown below.
+
+<img src="../images/ri-rdi-add-endpoint1.png">
+
+Enter your RDI server details in the dialog. The **RDI Alias** field can be any name you choose and it will be used as the primary name in the **RDI Instances** list view.
+
+<img src="../images/ri-rdi-add-endpoint2.png">
+
+You'll receive notification if your connection is successful.
+
+<img src="../images/ri-rdi-endpoint-added.png">
+
+## Create, test, and deploy RDI pipelines
+
+Begin by clicking the alias of your newly configured RDI connection in the **RDI Instances** view (for example, **Test connection** in the above image). You'll see the following dialog.
+
+<img src="../images/ri-rdi-pl-start.png">
+
+Choose from the following options:
+
+- **Download from server** - Download already-defined pipelines from your RDI configuration.
```

Comment:
"Download an already-defined pipeline" - currently, there is only one pipeline supported in RDI

Reply:
Done. Look for next merge.